CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changes by Version
 ==================
 
+0.44.0
```

Comment:
there seems to be missing items since the last release https://github.com/open-telemetry/opentelemetry-operator/commits/main

Reply:
Added to changelog